src/utils/download.ts

```diff
@@ -0,0 +1,25 @@
+import * as fs from 'fs';
+import path from 'path';
+import axios, { AxiosResponse } from 'axios';
+
+/**
+ * Downloads file from remote HTTP[S] host and puts its contents to the
+ * specified location.
+ * @param url URL of the file to download.
+ * @param destinationPath Path to the destination file.
+ */
+export async function download(url: string, filePath: string): Promise<string> {
+	const fileName = url.split('/').pop();
+	if (!fileName) {
+		throw new Error('Invalid remote path. No file name found.');
+	}
+	try {
+		const response: AxiosResponse = await axios.get(url, { responseType: 'arraybuffer' });
```

Comment:
Could be nice to have some progress logging to stderr

Reply:
I don't think we have any good examples of progress logging from axios. But see `onUploadProgress` in https://axios-http.com/docs/req_config for inspo. Not a blocker but a nice to have if you can whip it up quickly.

Reply:
Can also see how we do naive progress logging for uploads in ArFSDao in ardrive-core-js for inspo. Search for `progressCallback` there.

Reply:
You can use `responseType: 'stream'`, get the length with `content-length` and get the current length with `on('data', (chunk) => chunk.length)`

Something like that:
```js
const {response, headers} = await axios.get(url, { responseType: 'stream' });
const totalLength = headers['content-length'];

const fileWriter = fs.createWriteStream(
  path.resolve(__dirname, fileName)
)

data.on('data', (chunk) => updateProgress(chunk.length))
data.pipe(fileWriter)
```